### PR TITLE
added a footer

### PR DESCRIPTION
**Pull request by GubliAS**
hello

### CODE REVIEW

**Comment by KolDamer** [comment]
Right , I am about to review it

File: styles.txt [hunk 1]
```diff
@@ -2,4 +2,6 @@ this is my styling page.
 i love styling
 follow me\
 this is red and white styling please
+gubliiii
+this is the footer
```
Comment (KolDamer) [inline styles.txt:6]: Ohh **_the design_** wasn't really needed 
But it's fine anyways

File: styles.txt [hunk 2]
```diff
@@ -2,4 +2,6 @@ this is my styling page.
 i love styling
```
Comment (KolDamer) [inline styles.txt:2]: Yo i don't care

Comment (GubliAS) [reply styles.txt:2]: okay tell Gerald

**Pull request merged**